File: output/deepseek_chat/avatar/direct_translation/java/python/successful_translation/codeforces_231_A.py

t = int(input())
count = 0
while t > 0:
    a, b, c = map(int, input().split())
    if (a == 1 and b == 1) or (a == 1 and c == 1) or (b == 1 and c == 1) or (a == 1 and b == 1 and c == 1):
        count += 1
    t -= 1
print(count)

File: output/gpt_4o/avatar/algo_based_translation/python/java/successful_translation/codeforces_231_A.java

import java.util.Scanner;

public class codeforces_231_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n = scanner.nextInt();
        int c = 0;
        
        scanner.nextLine(); // Consume the remaining newline
        
        for (int i = 0; i < n; i++) {
            String[] input = scanner.nextLine().split(" ");
            int[] l1 = new int[input.length];
            
            for (int j = 0; j < input.length; j++) {
                l1[j] = Integer.parseInt(input[j]);
            }
            
            int sum = 0;
            for (int num : l1) {
                sum += num;
            }
            
            if (sum > 1) {
                c++;
            }
        }
        
        System.out.println(c);
        
        scanner.close();
    }
}

